glide3x/h3/glide3/src/fxglide.h

```diff
@@ -1884,7 +1884,7 @@ _grSstVRetraceOn(void);
 
 #ifdef __GNUC__
 
-extern __inline unsigned long getThreadValueFast (void)
+static __inline unsigned long getThreadValueFast (void)
```

Comment:
This should be OK.

Reply:
OK as in OK to go in, or OK as in "should work without changes"?

Reply:
The extern-to-static inline changes are OK to go in